inst/include/plant/patch.h

```diff
@@ -173,15 +212,15 @@ std::vector<double> Patch<T,E>::r_competition_effect_error(size_t species_index)
 
 template <typename T, typename E>
 void Patch<T,E>::compute_environment() {
-  if (parameters.n_residents() > 0) {
```

Comment:
deleting this is causing an issue creating an empty scm. I'm following up.

Reply:
leads to error "Error: Impossible bounds" when creating an empty sm in `test-scm.R`

Reply:
fixed in 635afd1d8857e6be7c4f174d9e3be631bf88b72e